src/state/selection/selectors/index.ts

```diff
@@ -1,12 +1,19 @@
 import { createSelector } from "reselect";
 import { reduce } from "lodash";
 import { UIDisplayData } from "@aics/simularium-viewer/type-declarations";
-import { SelectionEntry } from "@aics/simularium-viewer/type-declarations/simularium/SelectionInterface";
+import {
+    ColorChanges,
+    SelectionEntry,
+} from "@aics/simularium-viewer/type-declarations/simularium/SelectionInterface";
```

Comment:
you shouldn't be importing things from the file structure within the viewer. If you need these to be available to the front end, you should export them the same as `UIDisplayData` is above

Reply:
I'm a little confused by the way the type-declarations are generated and exported.

The interfaces for ColorChanges and UIDisplayData are declared in the same file (SelectionInterface), but the build scripts result in UIDisplayData being exported from @aics/simularium-viewer/type-declarations, and ColorChanges being exported from @aics/simularium-viewer/type-declarations/simularium.

I think that addresses your concern from above, as nothing is being imported directly from SelectionInterface, but I'm confused why the exports don't end up in the same place and Im not 100% sure that addresses your concern.

Reply:
did you add the export type to the viewer the same way it's done for `UIDisplayData`?

Reply:
you need to add it to the index file: 
```export type {
    SelectionStateInfo,
    UIDisplayData,
    VisDataFrame,
} from "./simularium";